request.go

```diff
@@ -981,3 +1001,84 @@ func (req *ExecuteRequest) Args(args interface{}) *ExecuteRequest {
 func (req *ExecuteRequest) Body(res SchemaResolver, enc *msgpack.Encoder) error {
 	return fillExecute(enc, req.expr, req.args)
 }
+
+type PrepareRequest struct {
```

Comment:
A description for the public API is missed.

Reply:
Added.